frontend/src/components/terminal.js

```diff
@@ -168,12 +35,14 @@ export default function Terminal() {
       >
         <div>Terminal</div>
       </div>
-      <div
-        className="h-screen w-full p-2"
-        style={{ backgroundColor: "#1E1E1E" }}
-      >
-        <pre id="terminal-text">{content}</pre>
-      </div>
+      <Editor
+        value={content}
+        readOnly="readonly"
+        onValueChange={() => {}}
+        highlight={() => highlight(content, "bash")}
```

Comment:
I don't think we should highlight using bash, just because the terminal will show program output which isn't really syntax highlightable